src/Compilers/CSharp/Portable/CSharpCodeAnalysis.csproj

```diff
@@ -36,788 +35,16 @@
-    <Compile Include="Declarations\DeclarationModifiers.cs" />
-    <Compile Include="Declarations\DeclarationTable.Cache.cs" />
-    <Compile Include="Declarations\DeclarationTable.cs" />
-    <Compile Include="Declarations\DeclarationTreeBuilder.cs" />
-    <Compile Include="Declarations\MergedNamespaceDeclaration.cs" />
-    <Compile Include="Declarations\MergedNamespaceOrTypeDeclaration.cs" />
-    <Compile Include="Declarations\MergedTypeDeclaration.cs" />
-    <Compile Include="Declarations\RootSingleNamespaceDeclaration.cs" />
-    <Compile Include="Declarations\SingleNamespaceDeclaration.cs" />
-    <Compile Include="Declarations\SingleNamespaceDeclarationEx.cs" />
-    <Compile Include="Declarations\SingleNamespaceOrTypeDeclaration.cs" />
-    <Compile Include="Declarations\SingleTypeDeclaration.cs" />
-    <Compile Include="DocumentationComments\DocumentationCommentIDVisitor.cs" />
-    <Compile Include="DocumentationComments\DocumentationCommentIDVisitor.PartVisitor.cs" />
-    <Compile Include="DocumentationComments\PEDocumentationCommentUtils.cs" />
-    <Compile Include="DocumentationComments\SourceDocumentationCommentUtils.cs" />
-    <Compile Include="Emitter\EditAndContinue\CSharpDefinitionMap.cs" />
-    <Compile Include="Emitter\EditAndContinue\CSharpLambdaSyntaxFacts.cs" />
-    <Compile Include="Emitter\EditAndContinue\EmitHelpers.cs" />
-    <Compile Include="Emitter\EditAndContinue\PEDeltaAssemblyBuilder.cs" />
-    <Compile Include="Emitter\EditAndContinue\CSharpSymbolMatcher.cs" />
-    <Compile Include="Emitter\Model\SourceAssemblySymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\ArrayTypeSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\AssemblyReference.cs" />
-    <Compile Include="Emitter\Model\AttributeDataAdapter.cs" />
-    <Compile Include="Emitter\Model\CustomModifierAdapter.cs" />
-    <Compile Include="Emitter\Model\DynamicTypeSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\EventSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\ExpandedVarargsMethodReference.cs" />
-    <Compile Include="Emitter\Model\FieldSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\GenericMethodInstanceReference.cs" />
-    <Compile Include="Emitter\Model\GenericNamespaceTypeInstanceReference.cs" />
-    <Compile Include="Emitter\Model\GenericNestedTypeInstanceReference.cs" />
-    <Compile Include="Emitter\Model\GenericTypeInstanceReference.cs" />
-    <Compile Include="Emitter\Model\MethodReference.cs" />
-    <Compile Include="Emitter\Model\MethodSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\ModuleReference.cs" />
-    <Compile Include="Emitter\Model\NamedTypeReference.cs" />
-    <Compile Include="Emitter\Model\NamedTypeSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\NamespaceSymbolAdapter.cs" />
-    <Compile Include="Emitter\Model\ParameterSymbolAdapter.cs">
+    <Compile Update="Emitter\Model\ParameterSymbolAdapter.cs">
```

Comment:
It had metadata (SubType=Code) originally and so this preserves that - however I just looked at the source and it doesn't seem to have any components. So not sure this is needed.